### PR TITLE
Replace Mapbox with OpenStreetMap

### PR DESCRIPTION
**Pull request by Falke-Design**
Because of https://github.com/Leaflet/Leaflet/issues/6336#issuecomment-954095682 I suggest to use OpenStreetMap tiles.

> You need permission for heavy use - is this heavy use? Who knows because that term is completely undefined which is why everybody that asks OWG just gets a shrug in response. That part of the policy is just stupid but nobody knows what else to say.

_Originally posted by @tomhughes in https://github.com/Leaflet/Leaflet/issues/6336#issuecomment-954095682_

Fix: #6336

### CODE REVIEW

**Review by mourner — CHANGES_REQUESTED** [review]
Tabs again please :( I'll some lint rules afterwards to check this.

**Comment by Falke-Design** [comment]
Fixed! Thank you for seeing this. I don't know why my IDE has the option "spaces to tabs" if it is not working ...

**Comment by johnd0e** [comment]
I like Mapbox tiles

**Comment by Malvoz** [comment]
> I like Mapbox tiles

With MapBox tiles the issue as described in https://github.com/Leaflet/Leaflet/issues/6336 remains:

> apparently the mapbox tokens are confusing more people.

File: docs/examples/quick-start/index.md [hunk 1]
```diff
@@ -42,28 +42,24 @@ Now you're ready to initialize the map and do some stuff with it.
...
 
 Note that the `setView` call also returns the map object --- most Leaflet methods act like this when they don't return an explicit value, which allows convenient jQuery-like method chaining.
 
-Next, we'll add a tile layer to add to our map, in this case it's a Mapbox Streets tile layer. Creating a tile layer usually involves setting the [URL template](/reference.html#tilelayer-url-template) for the tile images, the attribution text, and the maximum zoom level of the layer. In this example, we'll use the `mapbox/streets-v11` tiles from [Mapbox's Static Tiles API](https://docs.mapbox.com/api/maps/#static-tiles) (in order to use tiles from Mapbox, you must also [request an access token](https://www.mapbox.com/studio/account/tokens/)). Because this API returns 512x512 tiles by default (instead of 256x256), we will also have to explicitly specify this and offset our zoom by a value of -1.
+Next, we'll add a tile layer to add to our map, in this case it's a OpenStreetMap tile layer. Creating a tile layer usually involves setting the [URL template](/reference.html#tilelayer-url-template) for the tile images, the attribution text, and the maximum zoom level of the layer.
```
Comment (IvanSanchez) [inline docs/examples/quick-start/index.md:53]: I would like to see some kind of link to https://operations.osmfoundation.org/policies/tiles/ here, along the lines of "OSM tiles are fine for programming your Leaflet map, but read this if you're going to make heavy use of the tiles"

**Review by IvanSanchez — DISMISSED** [review]
Unfortunately this conflicts with the whole "let's move the website to a `/SlavaUkraini` path" thing from #8025. There's some renaming to do.

**Review by Malvoz — APPROVED** [review]
LGTM, there will be discrepancies between thumbnails for tutorials and actual tiles used but I think that's fine.

**Review by mourner — APPROVED** [review]

**Pull request merged**